src/index.html

```diff
@@ -11,7 +11,144 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+
+      <fieldset>
```

Comment:
Don't add empty lines between the parent and a child components
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
      method="get"
    >
      <fieldset>
```